### PR TITLE
Feature Offscreen Rendering

### PR DESCRIPTION
**Pull request by zimmermannubique**
- iOS & Shared Part

### CODE REVIEW

File: shared/src/map/MapScene.h [hunk 1]
```diff
@@ -91,4 +97,6 @@ class MapScene : public MapInterface, public SceneCallbackInterface, public std:
 
     bool isResumed = false;
     std::atomic_flag isInvalidated = ATOMIC_FLAG_INIT;
+
+    int64_t notReady;
```
Comment (stmitt) [inline shared/src/map/MapScene.h:101]: this flag is not used

File: shared/src/map/MapScene.cpp [hunk 2]
```diff
@@ -244,3 +248,52 @@ void MapScene::pause() {
             }
         }));
 }
+
+void MapScene::drawReadyFrame(const ::RectCoord & bounds, float timeout, const std::shared_ptr<MapReadyCallbackInterface> & callbacks) {
+
+    for (const auto &layer : layers) {
+        layer.second->enableAnimations(false);
```
Comment (stmitt) [inline shared/src/map/MapScene.cpp:255]: maybe we should re-enable the animation after the drawing?

File: shared/src/map/MapScene.h (hunk 1 above)
Comment (zimmermannubique) [reply shared/src/map/MapScene.h]: XX

File: shared/src/map/MapScene.cpp (hunk 2 above)
Comment (zimmermannubique) [reply shared/src/map/MapScene.cpp:257]: XX

**Review by stmitt — APPROVED** [review]

**Pull request merged**